std/hash/wyhash.zig

```diff
@@ -0,0 +1,99 @@
+const std = @import("std");
+const mem = std.mem;
+
+const primes = [_]u64{
+    0xa0761d6478bd642f,
+    0xe7037ed1a0b428db,
+    0x8ebc6af09c88c6e3,
+    0x589965cc75374cc3,
+    0x1d8e4e27c47d124f,
+};
+
+fn read_bytes(comptime bytes: u8, data: []const u8) u64 {
+    return mem.readVarInt(u64, data[0..bytes], @import("builtin").endian);
```

Comment:
Reading using native endianness of the target seems suspect to me. I do realise the reference just does a memcpy, but if one were to use this hash and serialize it somewhere it wouldn't be portable.

I think it'd be best to explicitly specify this as `.Little`. The Go and Rust implementations both do this, although the C# one doesn't.

Reply:
Good catch.